services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/settings.py

```diff
@@ -275,17 +275,19 @@ def _merge_resources_in_settings(
 
     # merge all resources
     empty_resource_entry: SimcoreServiceSettingLabelEntry = (
-        SimcoreServiceSettingLabelEntry(
-            name="Resources",
-            setting_type="Resources",
-            value={
-                "Limits": {"NanoCPUs": 0, "MemoryBytes": 0},
-                "Reservations": {
-                    "NanoCPUs": 0,
-                    "MemoryBytes": 0,
-                    "GenericResources": [],
+        SimcoreServiceSettingLabelEntry.parse_obj(
```

Comment:
as @pcrespov said, using parse_obj is always more secure than the constructor.

Reply:
not sure in this case makes a big difference though  :-)
I think it is secure when your data is in a ``dict``-like object but here we pass explicitly the values

Reply:
Anybody an idea why this fails stochastically?